authorize_test.go

```diff
@@ -1,68 +1,68 @@
-package cedar
+package cedar_test
```

Comment:
This is good hygiene. None of the tests in this file need to interact with any private bits in the package.